mava/utils/make_env.py

```diff
@@ -141,5 +180,7 @@ def make(config: DictConfig, add_global_state: bool = False) -> Tuple[Environmen
         return make_jumanji_env(env_name, config, add_global_state)
     elif env_name in jaxmarl.registered_envs:
         return make_jaxmarl_env(env_name, config, add_global_state)
+    elif env_name in _matrax_registry:
```

Comment:
```suggestion
    elif env_name == "Matrax":
```

Reply:
It seems like this change and the previous one to use `MatraxWrapper` directly make `_matrax_registry` completely unnecessary. Could we leave it out altogether?

Reply:
Ye I think leave it in for now, just like @RuanJohn said it keeps the pattern